Formula/aliyun-cli.rb

```diff
@@ -0,0 +1,35 @@
+class AliyunCli < Formula
+  desc "Universal Command-Line Interface for Alibaba Cloud"
+  homepage "https://github.com/aliyun/aliyun-cli"
+  url "https://github.com/aliyun/aliyun-cli/archive/v3.0.13.tar.gz"
+  sha256 "876c4ddc8c4129054196e6420735c600556710df03265cf72291840909a0d9e1"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["PATH"] = "#{ENV["PATH"]}:#{buildpath}/bin"
+    (buildpath/"src/github.com/aliyun/aliyun-cli").install buildpath.children
+    cd "src/github.com/aliyun/aliyun-cli" do
+      system "make", "metas"
+      system "go", "build", "-o", bin/"aliyun", "-ldflags", "-X 'github.com/aliyun/aliyun-cli/cli.Version=#{version}'", "main/main.go"
+      prefix.install_metafiles
+    end
+  end
+
+
```

Comment:
I would suggest to delete one empty line.

Reply:
fixed